### PR TITLE
[bitnami/apisix] Support for customizing standard labels

### PR DESCRIPTION
**Pull request by juan131**
### Description of the change

This PR follows up #18154 ensuring this chart is adapted to use new helpers' format, hence adding support for customizing standard labels.

### Benefits

User can customize standard labels.

### Possible drawbacks

None

### Applicable issues

N/A

### Additional information

#18154 must be merged **before** this PR

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/apisix/templates/dashboard/ingress.yaml [hunk 1]
```diff
@@ -9,20 +9,12 @@ kind: Ingress
...
-    {{- end }}
   {{- if or .Values.dashboard.ingress.annotations .Values.commonAnnotations }}
-  annotations:
-    {{- if .Values.dashboard.ingress.annotations }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.dashboard.ingress.annotations "context" $) | nindent 4 }}
-    {{- end }}
-    {{- if .Values.commonAnnotations }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
-    {{- end }}
+  {{- $annotations := merge .Values.controdashboardlPlane.ingress.annotations .Values.commonAnnotations }}
```
Comment (joancafom) [inline bitnami/apisix/templates/dashboard/ingress.yaml:16]: Typo
```suggestion
  {{- $annotations := merge .Values.dashboard.ingress.annotations .Values.commonAnnotations }}
```

File: bitnami/apisix/templates/dashboard/pdb.yaml
Comment (joancafom) [inline bitnami/apisix/templates/dashboard/pdb.yaml:1]: Is this intended?

Comment (juan131) [reply bitnami/apisix/templates/dashboard/pdb.yaml:1]: nope. fixing it

File: bitnami/apisix/templates/dashboard/ingress.yaml (hunk 1 above)
Comment (juan131) [reply bitnami/apisix/templates/dashboard/ingress.yaml]: fixed at https://github.com/bitnami/charts/pull/18456/commits/c898541be92c5f8545ad392a7358b3a420af6247

File: bitnami/apisix/templates/dashboard/pdb.yaml
Comment (juan131) [reply bitnami/apisix/templates/dashboard/pdb.yaml:1]: fixed at https://github.com/bitnami/charts/pull/18456/commits/c898541be92c5f8545ad392a7358b3a420af6247

File: bitnami/apisix/templates/dashboard/pdb.yaml [hunk 2]
```diff
@@ -3,30 +3,30 @@ Copyright VMware, Inc.
...
 
-{{- if .Values.dataPlane.pdb.create }}
+{{- if .Values.dashboard.pdb.create }}
 apiVersion: {{ include "common.capabilities.policy.apiVersion" . }}
 kind: PodDisruptionBudget
 metadata:
-  name: {{ template "apisix.data-plane.fullname" . }}
+  name: {{ template "apisix.dashboard.fullname" . }}
   namespace: {{ include "common.names.namespace" . | quote }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
```
Comment (joancafom) [inline bitnami/apisix/templates/dashboard/pdb.yaml:12]: You have not adapted to the new helper's format here.

File: bitnami/apisix/templates/dashboard/pdb.yaml [hunk 3]
```diff
@@ -3,30 +3,30 @@ Copyright VMware, Inc.
...
+  {{- if .Values.dashboard.pdb.minAvailable }}
+  minAvailable: {{ .Values.dashboard.pdb.minAvailable }}
   {{- end }}
-  {{- if .Values.dataPlane.pdb.maxUnavailable }}
-  maxUnavailable: {{ .Values.dataPlane.pdb.maxUnavailable }}
+  {{- if .Values.dashboard.pdb.maxUnavailable }}
+  maxUnavailable: {{ .Values.dashboard.pdb.maxUnavailable }}
   {{- end }}
   selector:
     matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
```
Comment (joancafom) [inline bitnami/apisix/templates/dashboard/pdb.yaml:29]: Neither here. You should also add the `podLabels`.

Comment (juan131) [reply bitnami/apisix/templates/dashboard/pdb.yaml]: Good catch! Done at https://github.com/bitnami/charts/pull/18456/commits/1fed0db0cec55398dcb476839ef72ba090a30091

File: bitnami/apisix/templates/dashboard/pdb.yaml (hunk 2 above)
Comment (juan131) [reply bitnami/apisix/templates/dashboard/pdb.yaml]: Good catch! Done at https://github.com/bitnami/charts/pull/18456/commits/1fed0db0cec55398dcb476839ef72ba090a30091

**Review by joancafom — APPROVED** [review]
LGTM! Thanks for the changes 🚀

**Pull request merged**